components/scroller/scroller.stories.js

```diff
@@ -0,0 +1,102 @@
+import DtScroller from './scroller';
+import BaseScrollerMdx from './scroller.mdx';
+import ScrollerDefault from './scroller_default.story';
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+export const argTypesData = {
+  default: {
+    control: { type: null },
+    description: 'Slot for main content',
```

Comment:
It is not clear what you are supposed to put into the default slot.

Reply:
Mhmm.. what about

```
description: 'Slot for main content that will be rendered inside the scroller.' +
      'Default scoped slot props are:\n\n ' +
      'item: item being rendered in a view.\n\n' +
      'index: reflects each items position in the items array\n\n' +
      'active: whether or not the view is active.',
```

Reply:
definitely better, but it's not totally clear what some of these properties really mean.

item - this is the data for an item? like a javascript object containing properties for the item?
index - this is obvious so it's fine
active - what does the view being active mean? is this like enabled vs disabled?

A dev with zero knowledge of the library this is using should be able to understand how to use this just by reading it

Reply:
Item is the item itself... it will be an object if you are using an array of objects
active return if the item is visible for the user or not


![image](https://user-images.githubusercontent.com/89984179/224758380-359b4fc2-19f1-411d-b645-501efff787b4.png)

Reply:
> I could update it to sth like:
> 
> 'item: the item itself, it will be an object if your list is an array with objects.\n\n' + 'index: reflects each items position in the list array\n\n' + 'active: return if the item is visible for the user or not.',

Definitely the right idea. Just to refine the wording a bit..

we can use jsdoc for types:

### default slot

Markup to display each item from the items array.

**item {*}:** the item currently being rendered from within your `items` array.
**index {number}:** the index of this item in the `items` array.
**active {boolean}:** whether this item is currently visible to the user.